tests/core/repository/mocks.py

```diff
@@ -24,13 +24,18 @@
 from src.taipy.core._version._version_manager import _VersionManager
 from taipy.config.config import Config
 
-Base = declarative_base()
+
+class Base:
+    __allow_unmapped__ = True
+
+
+Base = declarative_base(cls=Base)  # type: ignore
```

Comment:
Reference: https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#migration-to-2-0-step-six-add-allow-unmapped-to-explicitly-typed-orm-models